core/src/main/scala/filodb.core/memstore/PartKeyTantivyIndex.scala

```diff
@@ -153,19 +196,81 @@ class PartKeyTantivyIndex(ref: DatasetRef,
   }
 
   override protected def addIndexedField(key: String, value: String): Unit = {
-    ???
+    val buffer = docBufferLocal.get()
+
+    // 1 - indexed field
+    buffer += 1
+    TantivyQueryBuilder.writeStringToBuffer(key, buffer)
```

Comment:
Usage of `TantivyQueryBuilder` in write execution path is a bit misleading to the reader. It is not WYSIWYG . Class name could be renamed or refactored

Reply:
Yep, will name this better.